spec/models/trip_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Trip, type: :model do
+  context 'column presence and type' do
+    it { is_expected.to have_db_column(:id) }
```

Comment:
I'm not sure if I like those specs. I would personally omit if you're planning to cover the application with `e2e request` specs.